hilti/runtime/include/types/enum.h

```diff
@@ -29,5 +31,38 @@ bool has_label(const T& t, const TypeInfo* ti) {
 
     return false;
 }
+
+/**
+ * Converts a signed integer value into an enum value. The value does
+ * not need to correspond to a valid label. (Internally, this is a
+ * straight-forward cast.)
+ *
+ * @param t numerical value to convert
+ * @tparam T enum type
+ */
+template<typename T>
+T from_int(int64_t n) {
+    return static_cast<T>(n);
```

Comment:
Could we add a check whether the [underlying type of the enum](https://en.cppreference.com/w/cpp/types/underlying_type) can represent the value? We probably should also `static_assert` that `T` [is an enum](https://en.cppreference.com/w/cpp/types/is_enum).

Reply:
Our enums are always 64-bit signed underneath. Are you thinking in case this got used with something else that codegen'd enums?

Reply:
I think this would be a good sanity check to add while we are at it for future-proofing. I wouldn't expect this to produce an error right now.

Reply:
ok, will do